helix-term/src/application.rs

```diff
@@ -82,7 +82,7 @@ impl Application {
 
         let mut editor = Editor::new(size, theme_loader.clone(), syn_loader.clone());
 
-        let editor_view = Box::new(ui::EditorView::new(std::mem::take(&mut config.keys)));
+        let editor_view = Box::new(ui::EditorView::new(std::mem::take(&mut config.keys), config.terminal.middle_click_paste));
```

Comment:
I think this can be changed to a struct. Will do in #554, as there is need in another EditorView config. So, this will need to go into main branch after #554.

Reply:
I'm going to merge `master` here soon, this got fixed in a2ccfff